acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/node/besu/BesuNodeFactory.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.tests.dsl.node.besu;
+
+import java.net.URL;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import com.google.common.io.Resources;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class BesuNodeFactory {
+  private static final Logger LOG = LogManager.getLogger();
+  private static final Path runDirectory = Path.of(System.getProperty("user.dir"));
+  private static final Path besuInstallDir =
+      Path.of(System.getProperty("besuInstallDir", runDirectory.resolve("build/besu").toString()));
+  private static final Path executablePath = besuInstallDir.resolve("bin/besu");
+
+  public static BesuNode create(final BesuNodeConfig config) {
+    checkBesuInstallation();
+
+    final List<String> params = new ArrayList<>();
+    params.add(executablePath.toString());
+
+    if (config.getGenesisFile().isEmpty()) {
+      params.add("--network");
+      params.add("DEV");
+    } else {
+      params.add("--genesis-file");
+      params.add(Resources.getResource(config.getGenesisFile().get()).getPath());
+    }
+    params.add("--data-path");
+    params.add(config.getDataPath().toString());
+    params.add("--logging");
+    params.add("DEBUG");
+    params.add("--miner-enabled");
+    params.add("--miner-coinbase");
+    params.add("1b23ba34ca45bb56aa67bc78be89ac00ca00da00");
+    params.add("--host-whitelist");
+    params.add("*");
+    params.add("--p2p-port");
+    params.add("0");
+    params.add("--rpc-http-enabled=true");
+    params.add("--rpc-http-port");
+    params.add("0");
+    params.add("--rpc-http-host");
+    params.add(config.getHostName());
+    params.add("--rpc-ws-enabled=true");
```

Comment:
shouldn't need ws enabled

Reply:
it was enabled in the previous implementation, hence I left it as it is.